docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -0,0 +1,966 @@
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: 0730-azurefile-1597391432
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: azurefile
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+Use the `kubectl` command to create the persistent volume and persistent volume claim.
+
+```bash
+$ kubectl apply -f ~/azure/weblogic-on-aks/pv.yaml
+$ kubectl apply -f ~/azure/weblogic-on-aks/pvc.yaml
+```
+
+You will see the following output after each command, respectively.
+
+```bash
+$ kubectl apply -f ~/azure/weblogic-on-aks/pv.yaml
+persistentvolume/0730-azurefile-1597391432 created
+$ kubectl apply -f ~/azure/weblogic-on-aks/pvc.yaml
+persistentvolumeclaim/0730-azurefile-1597391432 created
+```
+
+Use the following command to verify:
+
+```bash
+$ kubectl get pv,pvc
+```
+
+Example output:
+
+```bash
+$ kubectl get pv,pvc
+NAME                                          CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                STORAGECLASS   REASON   AGE
+persistentvolume/0730-azurefile-1597391432   5Gi        RWX            Retain           Bound    default/0730-azurefile-1597391432   azurefile               16m
+
+NAME                                               STATUS   VOLUME                       CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/0730-azurefile-1597391432   Bound    0730-azurefile-1597391432   5Gi        RWX            azurefile      16m
+```
+
+> **Note**: Carefully inspect the output and verify it matches the above. `ACCESS MODES`, `CLAIM`, and `STORAGECLASS` are vital.
+
+#### Install WebLogic Server Kubernetes Operator into the AKS cluster
+
+The Oracle WebLogic Server Kubernetes Operator is an adapter to integrate WebLogic Server and Kubernetes, allowing Kubernetes to serve as a container infrastructure hosting WLS instances.  The operator runs as a Kubernetes Pod and stands ready to perform actions related to running WLS on Kubernetes.
+
+Kubernetes Operators use [Helm](https://helm.sh/) to manage Kubernetes applications. The operator’s Helm chart is located in the `kubernetes/charts/weblogic-operator` directory. Please check the Helm version first if you are using the Azure Cloud Shell, and run the corresponding command. This sample was tested with v3.0.0.
+
+```bash
+# Check the helm version
+$ helm version
+
+# For Helm 3.x, run the following:
+$ helm repo add weblogic-operator https://oracle.github.io/weblogic-kubernetes-operator/charts
+$ helm repo update
+$ helm install weblogic-operator weblogic-operator/weblogic-operator --version "3.0.0"
+
+# For helm 2.x, run the following:
```

Comment:
For helm 2.x, run the following: -> I think that you can remove these commands because support for Helm 2.x was removed in operator 3.0, https://oracle.github.io/weblogic-kubernetes-operator/release-notes/